myosuite/utils/__init__.py

```diff
@@ -17,4 +19,21 @@ def import_gym():
         raise ModuleNotFoundError(help)
     return gg
 gym = import_gym()
+    
+
+# utility to allow different numpy versions
+class NPRandomVersionWrapper:
+    def __init__(self, np_random):
+        self.np_random = np_random
 
+    def __getattr__(self, name):
+        if name == 'integers':
+            def integers(*args, **kwargs):
```

Comment:
If we know the version of numpy that has this issue, we might consider adding a [implemented for](https://github.com/MyoHub/myosuite/blob/e908da72dffab5192e7530a2a58147c94af7808f/myosuite/__init__.py#L11) condition like we do for gym

``` python
@implement_for("numpy", None, "0.13")
def np_random_integers():
    return self.np_random.integers(*args, **kwargs)

@implement_for("numpy", "0.13", None)
def np_random_integers():
    return self.np_random.randint(*args, **kwargs)

.....
```